audit_log/models.py

```diff
@@ -1,9 +1,12 @@
 from django.db import models
 from django.db.models import JSONField
+from django.utils.translation import gettext_lazy as _
 
 
 class AuditLog(models.Model):
     message = JSONField()
+    is_sent = models.BooleanField(default=False, verbose_name=_("is sent"))
```

Comment:
What about using a nullable DatetimeField (like `sent_at`) for this, which would hold a bit more perhaps useful information for debugging etc with ~zero cost?

Reply:
Good idea, I changed is_sent to sent_at and set the field to DateTimeField